src/scrabble.js

```diff
@@ -1,6 +1,305 @@
+      return tripleScoreLettersToRemove.indexOf(item) === -1
+    })
+    return arrayMinusDoubleTripleScoredLetters
+  }
+
+  // // REMOVE BRACKETS FROM ARRAY
+
+  cleanArrayOfBrackets () {
+    const cleanArrayForScoring = [...this.arrayMinusDoubleTripleScoredLetters].filter(item => {
+      return this.bracketsToRemove.indexOf(item) === -1
+    })
+
+    return cleanArrayForScoring
+  }
+
+  // single score
+  singleScore () {
+    const singleScore = this.arrayMinusDoubleTripleScoredLetters.map(letter => scoreObject[letter.toUpperCase()]).reduce((x, y) => x + y, 0)
+    console.log(singleScore)
+    return singleScore
+  }
+
+  // double score
+  doubleScore () {
+    const doubleScore = this.doubleScoreCurlyBrackets().map(letter => scoreObject[letter.toUpperCase()]).reduce((x, y) => x + y, 0)
+    console.log(doubleScore)
+    return doubleScore * 2
+  }
 
+  // triple score
+  tripleScore () {
+    const tripleScore = this.tripleScoreSquareBrackets().map(letter => scoreObject[letter.toUpperCase()]).reduce((x, y) => x + y, 0)
+    console.log(tripleScore)
+    return tripleScore * 3
+  }
+
+  scoresTotaled () {
+    this.totalScore = singleScore + doubleScore + tripleScore
+    console.log('.............', this.totalScore)
+    return this.totalScore
+  }
+}
+// -------------------------------------24-24-8-8-24-1-1-1-1-3-4-1
+const doubleScoreWord = new Scrabble('[X][X]{Y}{Y}[X]aaaapwu')
+const final = new Scrabble('xx')
+console.log('removeDoubleScoredLetters', final.removeDoubleScoredLetters())
+console.log('removeTripleScoredLetters', final.removeTripleScoredLetters())
+const wordArray = doubleScoreWord.wordArray()
+console.log('wordArray', wordArray)
+const scoredLettersDouble = doubleScoreWord.doubleScoreCurlyBrackets()
+const scoredLettersTriple = doubleScoreWord.tripleScoreSquareBrackets()
+console.log('scoredLettersDouble', scoredLettersDouble)
+console.log('scoredLettersTriple', scoredLettersTriple)
+
+// remove brackets from array
+
+const bracketsToRemove = ['[', ']', '{', '}']
+const arrayCleanOfBrackets = wordArray.filter(item => {
+  return bracketsToRemove.indexOf(item) === -1
+})
+console.log('arrayCleanOfBrackets.', arrayCleanOfBrackets)
+
+// remove double score letters from original array
+
+const arrayCleanOfDouble = arrayCleanOfBrackets.filter(item => {
+  return scoredLettersDouble.indexOf(item) === -1
+})
+console.log('arrayCleanOfDouble....', arrayCleanOfDouble)
+
+// remove triple score letters
+
+const arrayCleanDoubleTriple = arrayCleanOfDouble.filter(item => {
+  return scoredLettersTriple.indexOf(item) === -1
+})
+console.log('arrayCleanDoubleTriple', arrayCleanDoubleTriple)
+
+const scoreObject = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
```

Comment:
is it necessary to write these out again? this was already done on line 21 `this.scoresObject` to line 47, is there a benefit to writing them out again?